framework/Distributions.py

```diff
@@ -1804,13 +1804,6 @@ def getInputSpecification(cls):
                             is returned to $N$). In case the ``with replacement'' strategy is used, the distribution samples
                             always from the complete set of specified $N$ values.
                             """
-    lb = InputData.parameterInputFactory('lowerBound', contentType=InputTypes.FloatType, printPriority=109,
```

Comment:
@joshua-cogliati-inl I think instead of removing these lines of code, I could use that `removeSubWithName()` method you mentioned in chat. Let me know your thoughts.

Reply:
I'm curious about why we can remove the upper and lower bounds for these distributions; is it because it's defined in the base class?

Reply:
@PaulTalbot-INL Yes, I believe that's why we're getting those `INPUT SPEC ERROR` when running HERON before. When I removed the upper and lower bound for these distributions the errors went away.